src/physics/land/land.jl

```diff
@@ -15,14 +15,14 @@ export LandModel
 end
 
 # default constructor
-function LandModel(SG::SpectralGrid)
+#=function LandModel(SG::SpectralGrid)
     return LandModel(
         SeasonalLandTemperature(SG),
         SeasonalSoilMoisture(SG),
         VegetationClimatology(SG),
         NoRivers(SG),
     )
-end
+end =#
```

Comment:
I removed/commented off this constructor. The warning with the double definition of the constructor stops the pre-compilation which in turn messes with the extensions. Without the extensions working properly the differentiability tests also don't work